lib/plugins/features/movies/controller.js

```diff
@@ -27,3 +30,19 @@ exports.list = (query) => {
   })
   .fetchAll();
 };
+
+exports.addLocation = (payload) => {
+  return Bluebird.all([
+    new Location({ id: payload.location_id }).fetch(),
+    new Movie({ id: payload.movie_id }).fetch()
+  ])
+  .then((result) => {
```

Comment:
There is actually a Bluebird helper function [`.spread`](http://bluebirdjs.com/docs/api/spread.html) that is really useful to use right after a `.all`. If the resolved value is an array (which it always will be for `.all`), you can call a `.spread` instead of a `.then` to break out the array into separate arguments instead of one array.